### PR TITLE
Bug fixes for Release V1.6.4

### PR DESCRIPTION
**Pull request by vaishaliagola27**

### CODE REVIEW

File: admin/classes/rtbiz-hd-support/class-rtbiz-hd-support.php [hunk 1]
```diff
@@ -506,29 +501,15 @@ public function submit_request() {
...
-//				echo '</div>';
-//			} else {
-//				echo '<div class="rtbiz_hd-error">';
-//				echo '<p>' . __( 'Your server failed to send an email.', RTBIZ_HD_TEXT_DOMAIN ) . '</p>';
-//				echo '<p>' . __( 'Kindly contact your server support to fix this.', RTBIZ_HD_TEXT_DOMAIN ) . '</p>';
-//				echo '<p>' . sprintf( __( 'You can alternatively create a support request <a href="%s">here</a>', RTBIZ_HD_TEXT_DOMAIN ), 'https://rtcamp.com/premium-support/' ) . '</p>';
-//				echo '</div>';
-			}
-//			die();
+			$support_email = 'support@rtcamp.com';
```
Comment (milindmore22) [inline admin/classes/rtbiz-hd-support/class-rtbiz-hd-support.php:511]: There seems to be a hardcoded email Id for support email?

File: admin/classes/rtbiz-hd-support/class-rtbiz-hd-support.php [hunk 2]
```diff
@@ -452,13 +450,10 @@ public function submit_request() {
...
                                     </tr>
                                     <tr>
-                                        <td>Website</td><td>' . strip_tags( $form_data['website'] ) . '</td>
-                                    </tr>
-                                    <tr>
-                                        <td>Subject</td><td>' . strip_tags( $form_data['subject'] ) . '</td>
+                                        <td>Details</td><td>' . strip_tags( $form_data['details'] ) . '</td>
                                     </tr>
                                     <tr>
-                                        <td>Details</td><td>' . strip_tags( $form_data['details'] ) . '</td>
```
Comment (milindmore22) [inline admin/classes/rtbiz-hd-support/class-rtbiz-hd-support.php:461]: did we remove the details field?

File: admin/classes/rtbiz-hd-email-notification/class-rtbiz-hd-email-notification.php [hunk 3]
```diff
@@ -250,7 +250,10 @@ public function notification_new_followup_added( $comment, $comment_type, $uploa
 			}
 
 			$bccemails = $this->exclude_author( $bccemails, $comment->comment_author_email );
-			$bccemails  = apply_filters( 'rtbiz_hd_filter_adult_emails', $bccemails, $comment->comment_post_ID );
+
+			// Get unique emails.
+			$bccemails = array_map( 'unserialize', array_unique( array_map( 'serialize', $bccemails ) ) );
```
Comment (milindmore22) [inline admin/classes/rtbiz-hd-email-notification/class-rtbiz-hd-email-notification.php:255]: can you check if can use WP function `maybe_unserialize` and `maybe_serialize`

File: admin/classes/rtbiz-hd-support/class-rtbiz-hd-support.php (hunk 1 above)
Comment (vaishaliagola27) [reply admin/classes/rtbiz-hd-support/class-rtbiz-hd-support.php:511]: @milindmore22 

It is same from the start. 

https://github.com/rtCamp/rtbiz-helpdesk/commit/7ea9b7ae65ff8c4a29e23950c868778964f45a53#diff-2b419e1f76958d640efa051d8e7d92e1a2453ab985e595ab40b4f4d2a2370ea2

File: admin/classes/rtbiz-hd-support/class-rtbiz-hd-support.php (hunk 2 above)
Comment (vaishaliagola27) [reply admin/classes/rtbiz-hd-support/class-rtbiz-hd-support.php:461]: @milindmore22 

This change is from here https://github.com/rtCamp/rtbiz-helpdesk/pull/108/commits/eceb80df0147aa3c074ff8880488f46044e8469b

I'm not sure about the history of the change.

File: admin/classes/rtbiz-hd-email-notification/class-rtbiz-hd-email-notification.php (hunk 3 above)
Comment (vaishaliagola27) [reply admin/classes/rtbiz-hd-email-notification/class-rtbiz-hd-email-notification.php:255]: @milindmore22 

I think i've tried that change also and there was some issue. That's why kept as above, which is working perfectly fine.

**Pull request merged**